src/bubble_sort.c

```diff
@@ -0,0 +1,43 @@
+#include <stdbool.h>
+#include <stdio.h>
+
+#define SIZEOF(arr) sizeof(arr) / sizeof(*arr)
+
+void swap(int* xp, int* yp) {
+    int temp = *xp;
+    *xp = *yp;
+    *yp = temp;
+}
+
+void printArray(int* arr, int size) {
+	int i;	
```

Comment:
Why you have moved the declaration of the variable from the loop definition?

Reply:
done